app/services/qa/linked_data/graph_service.rb

```diff
@@ -0,0 +1,78 @@
+# Extend the RDF graph to include additional processing methods.
+module Qa
+  module LinkedData
+    class GraphService
+      attr_reader :graph
+
+      # Retrieve linked data from specified url
+      # @param [String] url from which to retrieve linked data
+      # @param [String | Symbol | Array<String|Symbol>] language for filtering graph (e.g. "en" or :en or ["en", "fr"] or [:en, :fr])
+      # @returns [RDF::Graph] graph of linked data
+      def initialize(url:, language: nil, remove_blanknode_subjects: false)
+        @graph = RDF::Graph.load(url)
+        filter(language: language, remove_blanknode_subjects: remove_blanknode_subjects)
```

Comment:
@elrayle I would either remove the filter here, or remove the filter method.  I think I would prefer removing it here, but I could go either way.  If you remove it here you could have the same graph filtered many ways.  I think having both is a bit confusing...

Reply:
I removed filter from the initializer.